CONTRIBUTING.md

```diff
@@ -268,20 +268,27 @@ also run successfully. Before you open a new pull request for your change,
 you'll want to run the test suite locally.
 
 The easiest way to run the test suite is to use
-[**tox**](https://tox.readthedocs.io/en/latest/#). You can install tox
-with pip: `pip install -U tox`. Tox provides several advantages, but the
-biggest one is that it builds an isolated virtualenv for running tests. This
-means it does not pollute your system python when running. Additionally, the
-environment that tox sets up matches the CI environment more closely and it
-runs the tests in parallel (resulting in much faster execution). To run tests
-on all installed supported python versions and lint/style checks you can simply
-run `tox`. Or if you just want to run the tests once run for a specific python
-version: `tox -epy37` (or replace py37 with the python version you want to use,
-py35 or py36).
+[**tox**](https://tox.readthedocs.io/en/latest/#). You can install tox with pip:
+`pip install -U tox`. Tox provides several advantages, but the biggest one is
+that it builds an isolated virtualenv for running tests. This means it does not
+pollute your system Python when running. Additionally, the environment that tox
+sets up matches the CI environment more closely and it runs the tests in
+parallel (resulting in much faster execution). In addition to the requirements
+installed by `tox` in the virtualenv, you'll need to install the LaTeX packages `xpic`,
+`qcircuit`, and `standalone`. These are typically provided in Linux distributions
+by packages of the same names prefixed by `texlive-`. For MacOS a good option is
+[MacTeX](https://www.tug.org/mactex/).
+
+To run tests on all installed
+supported Python versions and lint/style checks you can simply run `tox`.  At a
+minimum, you should run the lint/style checks and the tests for one version of
+Python. For example `tox -e lint && tox -epy311`. If you want to run the tests
+for a specific supported version of Python replace py311 with py37, py38, py39,
+or py310.
 
 If you just want to run a subset of tests you can pass a selection regex to
 the test runner. For example, if you want to run all tests that have "dag" in
-the test id you can run: `tox -epy37 -- dag`. You can pass arguments directly to
+the test id you can run: `tox -epy311 -- dag`. You can pass arguments directly to
```

Comment:
While we're updating these it might make more sense to just use `tox -epy` which just uses the current interpreter version implicitly instead of explicitly setting a version. This was my bias when I originally wrote this section because I have multiple interpreters installed and always explicitly specify which one I want to test with. But for most people reading this guide we don't need to specify a version.

Reply:
Fixed in 3c0ac1e90a3db7569412ae2f4b867d0b689384a4